### ISSUE TITLE
Add "Graph with Offsets" to Visual/Panel mode

### ISSUE THREAD

**Issue by Shadorain** [radareorg/radare2#18631]
## Description

I absolutely adore visual panel mode, it is pure brilliancy, but what would be incredibly nice would be to have the ability to either change the views of Graph panel (in visual panel mode) the way you can in Visual Graph mode with `p/P`, or simply just add a panel for Graph with Offsets which makes it easier to debug etc.
Also i noticed there is no Disassemble with Offsets either which should also be added I believe.

This setup would be perfect here imo with offsets able to be added
![image](https://user-images.githubusercontent.com/35006332/116000050-a6743380-a5bc-11eb-98b3-8820ad1d5654.png)

Would look like this but within a panel
![image](https://user-images.githubusercontent.com/35006332/116000069-bee44e00-a5bc-11eb-8474-5bcab8de5518.png)

I think y'all get the point haha. Thanks so much!

**Comment by radare** [comment]
just set graph.offset=true in your settings

**Comment by Shadorain** [comment]
Ooh awesome! I figured it was simple, should have looked there first instead of commands.
Do you know if there is a way to switch those views? (like in Visual Graph mode)

**Comment by Shadorain** [comment]
Im wondering if it could be done with some sort of alias/macro that you can press it visual mode that toggles `e graph.offset=...`
If that makes sense^

**Comment by trufae** [comment]
Yes this could be added easily, the main problem is deciding the right approach for panels where most keys are used for window manipualtion and such, any proposal to improve panels usability?

**Comment by trufae** [comment]
i think it makes sense to add this option in one of those two options:

* add a menu entry to toggle this option and refresh the panels containing a graph
* add another graph panel entry with this option set

This would be less keybinding disruptive at least..

**Comment by Shadorain** [comment]
> Yes this could be added easily, the main problem is deciding the right approach for panels where most keys are used for window manipualtion and such, any proposal to improve panels usability?

Sure i have an idea! 
Either 1: In Window mode (`w`) instead of having the normal Visual Pane `p/P` have in window mode it change the modes
Or 2: Click enter on the pane to enlarge it then open the `p/P` keys to it to change the modes (like in Visual Graph)

Would these be ideal ways of doing it? not sure, would need to be documentation to show this as well but I think it could work

**Comment by Shadorain** [comment]
Yeh so to your approaches 
- That would be interesting! Never thought of that, I was thinking more modal binds (i use too much vim lol)
- That was my original idea but I think everyone could benefit from many different changeable views instead of changing the entire panel setup (in turn making them resave their layout)

**Comment by radare** [comment]
Do you want to implement one of those proposals?
